prov/rxm/src/rxm_cq.c

```diff
@@ -1680,8 +1680,10 @@ void rxm_cq_write_error(struct util_cq *cq, struct util_cntr *cntr,
 void rxm_cq_write_error_all(struct rxm_ep *rxm_ep, int err)
 {
 	struct fi_cq_err_entry err_entry = {0};
+	struct rxm_coll_buf req = {0};
 	ssize_t ret = 0;
 
+	err_entry.op_context = &req;
```

Comment:
Something seems off in the call path if the code always expects op_context to reference some collective buffer structure.

Reply:
The rxm peer cq implementation was initially for collective offload only. See comments at L1883~L1889. If here we expect broader usage, some refactoring will be needed.

Reply:
Yeah I though so too. If rxm needs to write an error for peer then it feels like that should be done separately on that path and not in the peer cq path? I'm not really sure what the target for this write_error_all call is. Would it ever have a collective buffer?

Reply:
The changes were introduced by commit  93c1a52d2d6b11c82a78035265c00ed3366cbd07. Did we miss something there or do we forget some tricks that made it work?

Reply:
@shefty @j-xiong Updated to just revert the commit since it's not necessary. The revert should fix it since the error path since it goes through the normal util cq insert path. We can revisit the peer CQ later when/if we add peer CQ support outside of peer collectives.

Reply:
Thanks - can you also pick for v1.18.x?